### DISCUSSION TITLE
Errors trying to install pycuda

### DISCUSSION THREAD

**Discussion by jmpolinski** [inducer/pycuda#326] [Q&A]
I need pycuda to use with the program instaGRAAL (https://github.com/koszullab/instaGRAAL). 
I've tried installing pycuda with a tar file, with git clone, and with pip - all fail. Most failures end with:
```
error: command 'gcc' failed with exit status 1
make: *** [install] Error 1
```

I have attached a text file with all output for reference. 
[pycuda_install-output.txt](https://github.com/inducer/pycuda/files/6803620/pycuda_install-output.txt)

**Comment by inducer** [comment]
From your log:
```
src/cpp/mempool.hpp:105:7: error: ‘unique_ptr’ in namespace ‘std’ does not name a type
       std::unique_ptr<Allocator> m_allocator;
       ^
```
Your installation of gcc is too old. You need a version that supports at least rudimentary C++11.

**Comment by yunweilu** [comment]
I have exactly same issue with you, do you solve the problem?

**Comment by gouzidama** [comment]
gcc version 5.4 vim setup.py  extra_compile_args=["-std=c++11"]
![image](https://user-images.githubusercontent.com/49581223/143873475-b100e2a5-cfd8-4319-9b42-4afbbda1c494.png)

**Reply by Qcellaris** [reply]
Thanks for this suggestion! Although installation and running the test gave rise to a lot of warnings, it at least succeeded this time. I suppose I got pycuda up and running properly now.

**Reply by Yurri-hub** [reply]
Thanks a lot for your suggestion!!!
